src/style.css

```
@@ -1 +1,36 @@
-/* add styles here */
+
+
+body {
+  margin: 0;
+}
+
+
+
+.stars {
+  display: flex;
+}
+
+
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-image: url(./images/star.svg);
+}
+
+
```

Too many empty lines
```suggestion
body {
  margin: 0;
}

.stars {
  display: flex;
}

.stars__star {
  width: 16px;
  height: 16px;
  background-position: center;
  background-repeat: no-repeat;
  background-image: url(./images/star.svg);
}
```